config/storybook-preset-carbon/carbon-theme-addon/src/components/Panel.js

```diff
@@ -5,6 +5,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+'use strict';
```

Comment:
```suggestion
```

Feel free to ignore `'use strict'` if the module type is an ES Module versus CommonJS 👀